examples/standard/hw-settle/hw-settle.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: starlingx.windriver.com/v1
+kind: HostProfile
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: controller-profile
+  namespace: deployment
+spec:
+  hwSettle: 0
```

Comment:
Based on the example, what's the expected value of the hwSettle? "0" and "1"?

Reply:
Based on the example, expected value of hwSettle is "0"